page-connect.php

```diff
@@ -0,0 +1,65 @@
+        <p>All of the security associated with ownCloud are at work when you use the ownCloud Proxy service. ownCloud is still run by you, on your computer. You can monitor what is happening, manage accounts, check the logs, use your own SSL certificates, enable automatic updates and more. The appliance pre-configures much of ownCloud for you. All you have to do is start the appliance, complete the setup and log into the web URL it provides. The ownCloud browser wizard at the provided URL will step you through logging in for the first time. Of course, as you configure accounts for others to share on your ownCloud, we recommend you use secure and complex passwords for all accounts.</p>
+        <p>However, we want to be completely transparent about the nature of ownCloud Proxy. The ownCloud Proxy app is distributed and the source code is in the ownCloud repository for review in Github. The ownCloud Proxy service is provided by PageKite. All files transferred through the ownCloud Proxy service do pass through its data centers, and for a brief time are readable as the ownCloud Proxy service transfers incoming requests from SSL to SSH on the way to the private ownCloud Server. In theory, if someone wanted to, all files transferred to and from devices using the ownCloud Proxy service could – for that brief instant of time that they are being moved from the SSL terminated connection to the SSH tunnel in memory – read by someone with ownCloud Proxy service server access. While the risk is small, and our terms and conditions cover this situation, it is something we wanted to be clear about. However, there is no way for the ownCloud Proxy service to access your data on your private server, it can only broker a connection with your private server.</p>
+        </div>
+        <div class="col-md-12">
+        <h3>How are ownCloud Proxy and the ownCloud Server Appliance different than ownCloud?</h3>
+        <p>The ownCloud Server appliance contains the ownCloud server, an OS and a database, provided in a pre-configured, easy to download, fast to set up package. ownCloud Proxy is an app inside the ownCloud appliance that enables an admin to sign up for and manage use of the ownCloud Proxy Service. This service is simply a connection broker that connects users outside of a network firewall to an ownCloud instance behind the firewall, with no special configuration required.</p>
+        <p>Of course, if you already have ownCloud 8.1 up and running, the ownCloud Proxy app is available in <a href="http://apps.owncloud.com">the ownCloud app store</a> and can be installed easily in an existing ownCloud instance to achieve the same overall result.</p>
+        </div>
+        <div class="col-md-12">
+        <h3>What if I Want to Disconnect my ownCloud Proxy Account?</h3>
+        <p>If, at any time, you want to stop using ownCloud Proxy, simply disable the app in the ownCloud admin interface and it will stop working permanently.</p>
```

Comment:
Before uninstalling or disabling it you need to press the stop button in the GUI.